src/main/java/edu/harvard/h2ms/config/SecurityConfig.java

```diff
@@ -53,6 +55,11 @@ public AuthenticationManager authenticationManagerBean() throws Exception {
     return super.authenticationManagerBean();
   }
 
+  @Autowired
+  public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception{
+    //auth.inMemoryAuthentication().withUser("jqadams@h2ms.org").password("password").roles("ADMIN");

```

Comment:
Should this entire method get removed if it's not being used?

Reply:
I removed this method.

Reply:
It's still here, did you push your changes?